File: lib/scala/graph/jmcc-moto-rev.scala
// moto rev (jmcc) #1
val f = SinOsc.kr(0.2).mulAdd(10, 21)
val s = LFPulse.ar(f, Seq.apply(0, 0.1), 0.1)
RLPF.ar(s, 100, 0.1).clip2(0.4)

File: lib/hs/graph/jmcc-moto-rev.hs
-- moto rev (jmcc) #1
let f = sinOsc KR 0.2 0 * 10 + 21
    s = lfPulse AR f (mce2 0 0.1) 0.1
in clip2 (rlpf s 100 0.1) 0.4
